### PR TITLE
Move jvm pause metrics names

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?
Move jvm pause metrics names so that it can be used.

### Why are the changes needed?
This is for the jvm pause metrics name string reuse

### Does this PR introduce any user facing changes?
NA

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java [hunk 1]
```diff
@@ -728,17 +728,17 @@ public void removeBlocks(Collection<Long> blockIds, boolean delete) throws Unava
...
         if (!validator.apply(id)) {
           invalidBlocks.add(id);
         }
       }
     }
     if (!invalidBlocks.isEmpty()) {
-      long limit = 100;
       List<Long> loggedBlocks = invalidBlocks.stream().limit(limit).collect(Collectors.toList());
       LOG.warn("Found {} orphan blocks without corresponding file metadata.", invalidBlocks.size());
       if (invalidBlocks.size() > limit) {
```
Comment (maobaolong) [inline core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java:744]: If we put the over limit check to the while condition, this condition cannot be true forever?

Comment (yyongycy) [reply core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java]: you are right, this change is added by mistake. It is supposed to be part of another change.
Roll it back

**Review by maobaolong — APPROVED** [review]
LGTM, @yyongycy Thanks.

**Comment by maobaolong** [comment]
BTW, It would be better if we make all the metric name as constant.

**Comment by maobaolong** [comment]
@yyongycy But Feel free to do it by another ticket, for this PR, I think it is enough.

**Review by yuzhu — APPROVED** [review]

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Pull request merged**